packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -108,6 +109,7 @@ export interface OdspErrorResponse {
 
 /** Error encapsulating the error response from ODSP containing the redirect location when a resource has moved  */
 export class OdspRedirectError extends LoggingError implements IFluidErrorBase {
+	// eslint-disable-next-line import/no-deprecated
```

Comment:
Unfortunately, these have to remain as is because in `createOdspNetworkError` the `error` variable is typed as `IFluidErrorBase & OdspError`, and `OdspError` has stronger requirements for `errorType`.